### PR TITLE
48 add dummy data

### LINKED ISSUE #48: Fill database initially with dummy data [2]
For the next presentation to the client it would be nice to have some data in the database to show functionality. 5 entries (per table) will suffice; (the end goal is to integrate it together with the information scraping team so that we have real data, their team isn't confident that this will work until Friday)

### PR DESCRIPTION
**Pull request by Zyranix**
Wrote a seed file that fills the database with dummy data to work with.

Closes #48

### CODE REVIEW

**Comment by Zyranix** [comment]
@elenagensch We are not sure how and if we should data from your db model. Maybe you can give us some input, or @Margarete01 will discuss this with you in your meeting?

@tom-richter Do you need walls etc. for testing?

**Comment by tom-richter** [comment]
> 

I have just read the message by chance. Please contact me via Telegram. You do not have to create dummy data for our tables.

**Review by elenagensch — COMMENTED** [review]
Looks good. As Tom mentioned we don't need dummy data. Thanks for asking!

**Review by elenagensch — DISMISSED** [review]

**Review by PashPaul — DISMISSED** [review]

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,75 @@
...
+
+person_list.each do |email, first_name, last_name, title, image, status|
+  person_collection << Person.create(email: email, first_name: first_name, last_name: last_name, title: title, image: image,
+                                     status: status)
+end
+
+bundle = person_collection.zip chair_list
+
+bundle.each do |person, chair|
+  chair = Chair.create(name: chair, people: [person])
```
[the remark this answers is not in the document]
Comment (Zyranix) [reply db/seeds.rb:50]: You're right - we overlooked that :)

**Pull request merged**